kinetic/distribution.yaml

```diff
@@ -5487,6 +5487,20 @@ repositories:
       url: https://github.com/ros/nodelet_core.git
       version: indigo-devel
     status: maintained
+  non_persistent_voxel_layer:
+    doc:
+      type: git
+      url: https://github.com/SteveMacenski/nonpersistent_voxel_layer.git
+      version: master
+    release:
+      packages:
+      - nonpersistent_voxel_layer
+      url: https://github.com/SteveMacenski/nonpersistent_voxel_layer.git
```

Comment:
The release repository should be a separate repository. Please see the `bloom` tutorial for details.